game/src/edit/traffic_signals/edits.rs

```diff
@@ -57,9 +57,44 @@ impl ChangeDuration {
                     match signal.stages[idx].phase_type {
                         PhaseType::Fixed(_) => true,
                         PhaseType::Adaptive(_) => false,
+                        PhaseType::Variable(_, _, _) => false,
                     },
                 ),
             ]),
+            Widget::row(vec![Line("How long with no demand to end stage?")
+                .small_heading()
+                .draw(ctx)]),
+            Widget::row(vec![
+                "Seconds:".draw_text(ctx),
```

Comment:
I think it'd look slightly better with a `.centered_vert()` here and for the other two; mind adding it?